holoviews/plotting/bokeh/__init__.py

```diff
@@ -33,6 +33,9 @@
 
 Store.renderers['bokeh'] = BokehRenderer.instance()
 
+if len(Store.renderers) == 1:
```

Comment:
Guess I could but defining that renderer is literally the line above so it has to be bokeh.